docs/source/prereqs.md

```diff
@@ -172,10 +172,28 @@ Optional: Install the latest version of [jq](https://stedolan.github.io/jq/downl
 <!--- Indent entire section -->
 <div style="margin-left: 1.5em;">
 
-### Git
+### Docker
 
-Install the latest version of [git](https://git-scm.com/downloads) if it is not already installed.
-To use `Fabric binaries`, you will need to have the `uname` command available. You can get it as part of `Git` but beware that only the 64bit version is supported.
+Install the latest version of [Docker Desktop](https://docs.docker.com/get-docker/) if it is not already installed.
+
+### WSL2
+
+Both the fabric documentation and fabric samples rely heavily on a `bash` environment. The recommended
+path is to use WSL2 (Windows Subsystem for Linux version 2) to provide a native linux environment and then you can follow the linux pre-requisites section (excluding the linux docker pre-requisite as you already have docker Desktop) and install them into your WSL2 linux distribution.
+
+WSL2 may not be installed by default, you can check and install WSL2 by going into "Programs and Features", clicking on "Turn Windows features on or off" and ensuring that both "Windows Subsystem For Linux" and "Virtual Machine Platform" are ticked.
```

Comment:
- default - (not comma) (use dash - or semicolon;) 
- use "selected" not ticked